http.kafka.async.sasl.scram/README.md

```diff
@@ -0,0 +1,172 @@
+# http.kafka.async
+Listens on http port `8080` or https port `9090` and will correlate requests and responses over the `items-requests` and `items-responses` topics in Kafka, asynchronously.
```

Comment:
Update description to include the main idea, that we use SASL SCRAM authentication to Kafka.

Reply:
done